### PR TITLE
Update sap-hana version, add hwRevision field to HANA instance model

### PR DESCRIPTION
**Pull request by AprilZhaoWY**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/index.json [hunk 1]
```diff
@@ -1394,7 +1394,7 @@
                     "summary": "Additional commands for working with SAP HanaOnAzure instances.",
                     "version": "0.3.3"
                 },
-                "sha256Digest": "12f0f12d9d1f525b2f0cf34b05bfedb5eef87f5b554e80cf3689265726245f25"
+                "sha256Digest": "d1c76239372837386535ecaf226dd0677014e2f2520c1329dbd5fda456000d36"
```
Comment (williexu) [inline src/index.json]: What happened to the last wheel that was hosted? and if changes were made, why is the version not bumped????

Comment (AprilZhaoWY) [reply src/index.json]: We do not bump our versions every time there is a change. So this is just a minor change, I just updated the .whl file. Let me know if this is okay.

@PakDLiu  have we done this before?

Comment (williexu) [reply src/index.json]: @AprilZhaoWY Even minor changes should bump the patch version.

Comment (AprilZhaoWY) [reply src/index.json]: @williexu Got it, will make the change in our code and then update this PR

Comment (williexu) [reply src/index.json]: Thanks 👍

Comment (AprilZhaoWY) [reply src/index.json]: @williexu  Updated! Please take a look

**Pull request merged**